packages/app/src/runs/RunsLayout.vue

```diff
@@ -0,0 +1,154 @@
+<template>
+  <div
+    v-if="isUsingGit"
+    class="flex flex-col gap-[24px]"
+  >
+    <ul
+      data-cy="runsLayout-git"
+      class="flex flex-col gap-[16px] relative before:content-[''] before:absolute before:top-[20px] before:bottom-[10px] before:w-[2px] before:border-2 before:border-dashed before:border-l-0 before:border-y-0 before:border-r-gray-100 before:left-[7px]"
+    >
+      <li
+        v-for="sha of Object.keys(groupByCommit)"
+        :key="sha"
+        :data-cy="`commit-${sha}`"
+        :class="`${!groupByCommit[sha].runs ? 'mb-[-24px]' : ''}`"
```

Comment:
I think we could make this more readable by using conditional class syntax
```suggestion
        :class="{ 'mb-[-24px]': !groupByCommit[sha].runs }"
```

Reply:
Updated.